docs/stellarphot/getting_started.rst

```diff
@@ -0,0 +1,93 @@
+Getting started
+###############
+
+Installation
+============
+
+If you are testing a pre-release version of stellarphot we recommend setting up
+a virtual environment and installing stellarphot in this environment.
+
+Only use one of the methods below for making a virtual environment.
+
+Creating an environment with `conda` or `mamba` (use whichever one you have installed)::
+
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install --pre stellarphot
+
+Creating an environment with `virtualenv`::
+
+    python -m venv stellarphot-test
+    source stellarphot-test/bin/activate
+    pip install --pre stellarphot
+
+To install stellarphot without creating an environment, use::
+
+    pip install --pre stellarphot
+
+You can remove stellarphot with::
+
+    pip uninstall stellarphot
+
+Overview
+========
+
+You will go through this process to do photometry.
+
+#. You need to make some settings, like camera properties, observatory information, and passband maps. You may only need
+   to do this step once if you use the same equipment for all of your observations.
+#. Settings specific to an object need to be made:
+
+   #. night of data, like the photometry aperture radius, need to be made.
```

Comment:
I was confused by this line, I assume you mean settings specific to each night of data need to made in addition to those specific to an object...